src/main/java/com/dajudge/kindcontainer/KindContainer.java

```diff
@@ -108,14 +109,17 @@ public KindContainer(final KubernetesImageSpec<KindContainerVersion> imageSpec)
                     volumes.add(varVolume);
                     volumes.add(modVolume);
 
-                    final List<Bind> binds = new ArrayList<>(asList(cmd.getBinds() == null ? new Bind[]{} : cmd.getBinds()));
-                    binds.add(new Bind(volumeName, varVolume, true));
-                    binds.add(new Bind("/lib/modules", modVolume, ro));
+                    final HostConfig hostConfig = cmd.getHostConfig();
```

Comment:
I don't think this is related. Can we maybe refactor this into a separate PR?

Reply:
Sure, I'll drop the commit here.

Reply:
I'd still be very greatful to see this change as a separate maintenance PR if you can spare the time! :smile: